CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 0.55.0 / 2022-03-09
+
+* [CHANGE] Enabled read-only root filesystem for containers generated from the Prometheus, Alertmanager and ThanosRuler objects.
+* [CHANGE] Disabled privilege escalation for the containers generated from Prometheus, Alertmanager and ThanosRuler objects.
+* [CHANGE] Dropped all capabilities for the containers generated from Prometheus, Alertmanager and ThanosRuler objects.
+* [CHANGE] Enables 'readonlyRootFilesystem: true' for the Prometheus statefulset. #4566
```

Comment:
i would mention the changed behavior for QueryLog 

```suggestion
* [CHANGE] Added `emptyDir` volume to the Prometheus statefulset when `spec.queryLogFile` is only a base filename (e.g. `query.log` as opposed to `/tmp/query.log`). When the path contains a full path, a volume + volume mount should be explicitly given in the Prometheus spec since the root file system is now read-only. #4566
```